ALU_wrapper.py

```diff
@@ -0,0 +1,128 @@
+
+import os
+import glob
+import smtplib
+import argparse
+import subprocess
+from email.message import EmailMessage
+
+mail_to = ""  # Change this, and uncomment argument before EOF
+pythonpath = "/apps/bio/software/anaconda2/envs/hcp/bin/python3"  # since server (as for now) has python2 installed
+
+parser = argparse.ArgumentParser(description='Wrapper to find non-processed folders, run ALU_parser on them.')
+parser.add_argument("-p", "--path", help="Path to folder with sequence-subfolders.", action="store", default=".")
+# /seqstore/remote/share/crc
+parser.add_argument("-r", "--run", help="Run for real. If not specified, just dry-run without any changes.",
+                    action="store_true")
+parser.add_argument("-v", "--verbose", help="Show more information while running.", action="store_true")
+args = parser.parse_args()
+sendmail = False
+seqpath = args.path
+
+
+def email_general(msg_to, msg_from, subject, body, attachment=None):
+    msg = EmailMessage()
+    msg.set_content(f'{body}\n'
+                    f'\n'
+                    f'Kind regards,\n'
+                    f'Clinical Genomics Gothenburg')
+    msg['Subject'] = f'{subject}'
+    msg['From'] = msg_from
+    msg['To'] = msg_to
+    # msg['Cc'] = "clinicalgenomics@gu.se"
+    if attachment:  # Add attachment if provided
+        csv_filename = os.path.basename(attachment)
+        with open(attachment, 'rb') as f:
+            data = f.read()
+            msg.add_attachment(data, maintype='text', subtype='plain', filename=csv_filename)
+    s = smtplib.SMTP('smtp.gu.se')
+    s.send_message(msg)
+    s.quit()
+
+
+def vcfparse(updated_file, v):
+    # TODO: Rev this part / rewrite
+    new_vcf = open(updated_file, 'w')
+    with open(v, "r") as vcf:
+        info_list = []
+        for line in vcf:
+            if ("ALU" in line or "LINE1" in line) and not line.startswith('#'):
+                info = line.split(';')
+                matching = [s for s in info if "SVLEN" in s]
+                match = str(matching).split('=')
+                length = match[1]
+                pos = (line.split('\t'))[1]
+                end = int(pos) + int(length.strip("']"))
+                info.insert(1, f"END={end}")
+                info_list.append(';'.join(info))
+            else:
+                info_list.append(line)
+    new_vcf.write(''.join(str(e) for e in info_list))
+    new_vcf.close()
+
+
+def alurunner():
+    vcf_list = glob.glob(path + '/*SV*.vcf*')
+    print(vcf_list)
+    for v in vcf_list:
+        updated_file = path + '/' + os.path.splitext(os.path.basename(v))[0] + '_melt_updated' + '.vcf'
+        print(v)
+        vcfparse(updated_file, v)
+
+
+def sendtoslims():
+    # TODO: Connect, send, what to send?
+    pass
+
+
+def compressfiles(folderpath):
+    destination = seqpath + "/saved/"  # This needs changing. Now set to subfolder saved under input-path (seqpath).
+    filename = folderpath.split('/').pop() + ".tgz"
```

Comment:
could this be .tar.gz? Just to be more compatible with other files on the cluster

Reply:
this is a tar.gz... the naming convention tgz is DOS-friendly (you should only use a three-letter extension in the good old days), and a sign of my age. So that is just a question of changing the name of the output, easy peasy.

Reply:
yeah I see. Was just thinking that we usually use the .tar.gz extension in other places, so just a harmonisation thing.

Reply:
Changed to .tar.gz in commit 433c048